test/cpp/qps/qps_worker.h

```diff
@@ -39,11 +39,13 @@ extern std::vector<grpc::testing::Server*>* g_inproc_servers;
 class QpsWorker {
  public:
   explicit QpsWorker(int driver_port, int server_port,
-                     const std::string& credential_type);
+                     const std::string& credential_type,
+                     bool die_on_connection_failure);
   ~QpsWorker();
 
   bool Done() const;
   void MarkDone();
+  bool ShouldDieOnConnectionFailure();
```

Comment:
nit: `bool die_on_connection_failure() const` since this is a trivial getter

Reply:
Done.